ui/lib/core/addon/components/form-field.js

```diff
@@ -14,6 +14,7 @@ import { addToArray } from 'vault/helpers/add-to-array';
 import { removeFromArray } from 'vault/helpers/remove-from-array';
 import { isEmpty } from '@ember/utils';
 import { presence } from 'vault/utils/forms/validators';
+import { get } from '@ember/object';
```

Comment:
confirmed with @zofskeez that using `get` will still work with the new api service resources (Thank you!!)